CHANGELOG.develop

```diff
@@ -423,6 +424,16 @@ Benner and Paweł Siudak):
 - Renamed =spacemacs/mplist-get= function to =spacemacs/mplist-get-values=,
   renamed =spacemacs/plist-get= function to =spacemacs/mplist-get-value=, and
   refactored the latter function (thanks to Benjamin Reynolds)
+- Fixed pdumper enabled on ~SPC f e R~ no matter what (thanks to Sylvain Benner)
```

Comment:
Because dump suport is a new feature since the last release, fixes to this new feature by its original author (Sylvain) probably don't need an entry.